mir_eval/separation.py

```diff
@@ -328,7 +438,8 @@ def _safe_db(num, den):
     return 10 * np.log10(num / den)
 
 
-def evaluate(reference_sources, estimated_sources, **kwargs):
+def evaluate(reference_sources, estimated_sources,
+             framewise=False, window=30*44100, hop=15*44100, **kwargs):
```

Comment:
1. Since you are using `**kwargs` and `filter_kwargs`, you should not add `window` and `hop` as separate keyword arguments.  They already can be supplied thanks to `**kwargs`, and they will not be passed to functions which do not accept them thanks to `filter_kwargs`.

1. Why add a flag for computing framewise metrics or not?  The `mir_eval` convention is that `evaluate` calls all of the different evaluation functions, so in this case we should just call (and return scores for) both the framewise and non-framewise functions.